src/ui/flutter_app/lib/widgets/game_page.dart

```diff
@@ -885,118 +887,125 @@ class _GamePageState extends State<GamePage>
               onPressed: onAutoReplayButtonPressed,
             ),
             */
-            ],
-          ),
-        );
-      },
+          ],
+        ),
+      ),
     );
   }
 
-  onOptionButtonPressed() {
+  void onOptionButtonPressed() {
     Navigator.push(
       context,
       MaterialPageRoute(builder: (context) => GameSettingsPage()),
     );
   }
 
-  onMoveButtonPressed() {
+  void onMoveButtonPressed() {
```

Comment:
The reason for the modification of the source code of this function is unknown, and it is not easy to visualize from the code differences.